src/provisioner/config.py

```diff
@@ -58,9 +59,14 @@ def save(self, cortx_conf, cortx_solution_config):
         """ Save cortx-config into confstore """
 
         try:
-            cortx_conf.copy(cortx_solution_config)
+            cortx_solution_config_keys = []
+            for key in Conf.get_keys(cortx_solution_config):
+                if key.startswith('cortx'):
```

Comment:
This can be optimized to the following code

cortx_solution_config_keys = filter(lambda x: x.startswith('cortx'),Conf.get_keys(cortx_solution_config))

Reply:
Done.